docs/structure-samples/csharp/csharp.md

```diff
@@ -0,0 +1,53 @@
+## C# and .NET project folder structure
+
+The following guidelines and the folders included, represent the conventional structure for a C#/.NET standard application.
+
+```bash
+~root
+│
+├── .devcontainer.json          # Moved to top
+├── .github/
+│   └── workflows/
+│       ├── workflow1.yml
+│       ├── workflow2.yml
+│       └── ...                     
+│
+├── infra/
+│   ├── main.bicep
+│   ├── main.parameters.bicep
+│   ├── abbreviations.json
+│   └── ...                         
+│
+├── src/
```

Comment:
We may only need to check if /src folder exist. Even /test folder may be structured in different place. both under /src and under root are common practice.